### ISSUE TITLE
What is the correct command to reproduce the accuracy results in the paper?

### ISSUE THREAD

**Issue by jmgrn61** [tensorflow/models#5933]
**Have I written custom code: No
OS Platform and Distribution: Ubuntu 16.04 LTS
TensorFlow installed from: https://www.tensorflow.org
TensorFlow version: Tensorflow 1.8
Bazel version: NA
CUDA/cuDNN version: cudnn-8.0-linux-x64-v6.0
GPU model and memory: Tesla K80 and 24GB
Exact command to reproduce: python eval.py --dataset_dir=/Users/andy/attention_ocr/models/research/attention_ocr/python/datasets/data/fsns/ --split=test --batch_size=204 --num_batches 1000**

I believe many people will have similar question with mine.  According to the instruction in the readme file, I downloaded the pre-trained model "attention_ocr_2017_08_09.tar.gz", and I guess I should run eval.py to get the testing accuracy.  

However, the sequence accuracy easily got above 93% with this downloaded model. It seems the default implementation in this code, the num_batches is set to 100, so the accuracy 93% must be based on some subset of images in the training set.

So my question is, what is the exact command here to evaluate a model (i.e. attention_ocr_2017_08_09.tar.gz) on the whole test data and reproduce ~84% of accuracy?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by qwangku** [comment]
Have I written custom code 
OS Platform and Distribution: Ubuntu 16.04 LTS
TensorFlow installed from: https://www.tensorflow.org
TensorFlow version: Tensorflow 1.8
Bazel version: NA
CUDA/cuDNN version: cudnn-8.0-linux-x64-v6.0
GPU model and memory: Tesla K80 and 24GB
Exact command to reproduce: python eval.py --dataset_dir=/Users/andy/attention_ocr/models/research/attention_ocr/python/datasets/data/fsns/ --split=test --batch_size=204 --num_batches 1000

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
